skglm/tests/test_cv.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+import pytest
+from sklearn.datasets import make_regression
+from sklearn.linear_model import ElasticNet
+from sklearn.model_selection import GridSearchCV, KFold
+from skglm.datafits import Quadratic
+from skglm.penalties import L1_plus_L2
+from skglm.solvers import AndersonCD
+from skglm.cv import GeneralizedLinearEstimatorCV
+
+
+@pytest.mark.parametrize("seed", [0, 42])
+def test_elasticnet_cv_matches_sklearn(seed):
+    """Test GeneralizedLinearEstimatorCV matches sklearn GridSearchCV for ElasticNet."""
+    X, y = make_regression(n_samples=100, n_features=20, noise=0.1, random_state=seed)
+
+    alphas = np.array([0.001, 0.01, 0.1, 1.0])
```

Comment:
picking absolute values for alphas is dangerous ; scale them by `\Vert X.T @ y \Vert_\infty / n` which is the smallest alpha for which the solution is 0